.github/workflows/build.yml

```
@@ -51,7 +51,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v4
         with:
-            node-version: '18'
+            node-version: '20.14.0'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)